### DISCUSSION TITLE
Add support for MySQL Database

### DISCUSSION THREAD

**Discussion by dcousens** [keystonejs/keystone#7510] [Feature requests]
With Prisma supporting MySQL,  will, and or when will Keystone 6 be adding support?

**Comment by willemmulder** [comment]
Yes please! Note that this will also enable users to use PlanetScale, which is MySQL based and is gaining traction lately.

**Comment by dcousens** [comment]
Tracking this feature in https://github.com/keystonejs/keystone/pull/7538

**Reply by willemmulder** [reply]
Excellent!

**Comment by craftogrammer** [comment]
Yes, please add this feature!
